Controllers/Users.controller.js

```diff
@@ -0,0 +1,91 @@
+const usersdb=require('../Models/users.model');
+const jwt=require('jsonwebtoken');
+
+module.exports.signupUser=async (req,res)=>{
+    const {name,email,password}=req.body;
+    let data=null;
+    try{
+    if(await usersdb.findOne({email:email})){
+        return res.status(208).json({
+            ok:false,
+            message:"User Already exists in the system"
+        })
+    }
+    if(name && (email && new RegExp("^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+$").test(email)) && password){
+        data=await usersdb.create({
+            name:name,
+            email:email,
+            password:password
+        })
+    }
+    else{
+        return res.status(400).json({
+            ok:false,
+            message:"Invalid Request"
+        })
+    }
+    if(data){
+        return res.status(201).json({
+            ok:true,
+            message:"User Added Successfully"
+        })
+    }    
+    }catch(error){
+        console.log(error)
```

Comment:
remove console logs

Reply:
Hey kunal console logging errors are need for checking the logs if some error occurs